privval/signer_listener_endpoint.go

```diff
@@ -2,10 +2,11 @@ package privval
 
 import (
 	"fmt"
-	"github.com/Finschia/ostracon/privval/internal"
 	"net"
 	"time"
 
+	"github.com/Finschia/ostracon/privval/internal"
+
 	privvalproto "github.com/tendermint/tendermint/proto/tendermint/privval"
 
 	"github.com/Finschia/ostracon/libs/log"
```

Comment:
Would you like to combine the imports of ostracon into the same block?
```suggestion
	privvalproto "github.com/tendermint/tendermint/proto/tendermint/privval"

	"github.com/Finschia/ostracon/libs/log"
	"github.com/Finschia/ostracon/privval/internal"
```